Maple2.Server.Game/PacketHandlers/UserEnvHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Packets;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class UserEnvHandler : PacketHandler<GameSession> {
+    public override RecvOp OpCode => RecvOp.RequestUserEnv;
+
+    private enum Command : byte {
+        ChangeTitle = 1,
+    }
+
+    public override void Handle(GameSession session, IByteReader packet) {
+        var command = packet.Read<Command>();
+        switch (command) {
+            case Command.ChangeTitle:
+                HandleChangeTitle(session, packet);
+                break;
+        }
+    }
+
+    private void HandleChangeTitle(GameSession session, IByteReader packet) {
+        int titleId = packet.ReadInt();
+
+        if (titleId < 0 || !session.Player.Value.Unlock.Titles.Contains(titleId) && titleId != 0) {
```

Comment:
I think this can be simplified to:
```cs
titleId <= 0 || !session.Player.Value.Unlock.Titles.Contains(titleId)
```

but maybe just doing the Contains check is sufficient. Why would we be adding invalid titleIds to a character.

Reply:
A titleId of 0 is valid (No title). But I'll adjust so it only checks for 0 or contains.